yva-backend/src/main/resources/db/migration/V5.3__Lukio_lupa_migration.sql

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Eikö nämä datan muokkaukset voisi kokonaisuudessaan koota repeatable-migraatioon? Olettaen, että esitysmallin ja paatoskierroksen id:n ovat yksikäsittelisiä, niin insert-lauseet voisi olla muotoa `insert into ... on conflcit do nothing`

Reply:
Periaatteessa varmaan voisi, mutta tuleekohan muutosten seuraamisesta hankalampaa kun repeatable scripti alkaa paisua isoksi. Eli tämä versioitu migraatio scripti on kuitenkin helppo liittää per tiketti ja esim. merget tulee olemaan helpompaa kun tehdään myös näitä hotfix korjauksia.

Reply:
No joo. Develop alkaa olla aika kaukana hotfixista, jolloin merget ei menisi niin helposti.

Itse oon tottunut käyttämään tiketin id:tä commit messageissa, jolloin saa aika kätevästi kaikki tiketteihin liittyvät muutokset selville.

Reply:
Katsotaan miten tämä mergeily yms. etenee muokataan toimintamallia jos havaitaan suurempia ongelmia.